src/lib/__mocks__/mockApiRequests.ts

```diff
@@ -311,14 +328,26 @@ export async function mockFetch(
         );
     }
   }
+
   if (endpoint === 'ONBOARDING_POST_LEGALS') {
     switch (endpointParams.externalInstitutionId) {
       case 'error':
         return genericError;
       default:
-        return new Promise((resolve) =>
-          resolve({ data: undefined, status: 200, statusText: '200' } as AxiosResponse)
-        );
+        const certifiedUser: UserOnCreate | undefined = (
+          (data as any).users as Array<UserOnCreate>
+        ).find((u) => u.taxCode === 'XXXXXX00A00X000X');
+        if (
+          certifiedUser &&
+          certifiedUser.name === 'CERTIFIED_NAME' &&
+          certifiedUser.surname === 'CERTIFIED_SURNAME'

```

Comment:
both name and surname should be as the expected value to give a 200.
This condition is not working as expected